RecoBTag/SecondaryVertex/python/pfSecondaryVertexTagInfos_cfi.py

```diff
@@ -20,3 +20,6 @@
         extSVCollection     = cms.InputTag('secondaryVertices'),
         extSVDeltaRToJet    = cms.double(0.3)
 )
+
+pfSecondaryVertexTagInfos.trackSelection.pixelHitsMin = cms.uint32(1)
+pfSecondaryVertexTagInfos.trackSelection.totalHitsMin = cms.uint32(0)
```

Comment:
why not in vertexTrackSelection_cff ?
The legacy taggers (secondaryVertexTagInfos , the non-pf) use it.
If this is our new default, I'd rather see the default modified and if the legacy needs to have the old values, override them in the legacy definitions.